Mathlib/Algebra/Hom/Equiv/Basic.lean

```diff
@@ -444,6 +444,8 @@ theorem ext {f g : MulEquiv M N} (h : ∀ x, f x = g x) : f = g :=
 #align mul_equiv.ext MulEquiv.ext
 #align add_equiv.ext AddEquiv.ext
 
+attribute [ext] AddEquiv.ext
```

Comment:
This was not added automatically by `to_additive`. Is this a bug that should be reported?

Reply:
This will be solved in https://github.com/leanprover-community/mathlib4/pull/954.